pipelinewise/cli/pipelinewise.py

```diff
@@ -1320,7 +1320,7 @@ def stop_tap(self, sig=None, frame=None):
 
                 # Terminate all the processes in the current process' process group.
                 for child in parent.children(recursive=True):
-                    if os.getpgid(child.pid) == pgid and child.status == 'running':
+                    if os.getpgid(child.pid) == pgid and child.status() == 'running':
```

Comment:
can you write regression tests for this?

Reply:
checking status is not necessary, I removed it and now it is same as previous release and no need regression test!